src/Microsoft.Azure.SignalR/HubHost/ServiceHubDispatcher.cs

```diff
@@ -27,7 +28,18 @@ internal class ServiceHubDispatcher<THub> where THub : Hub
         private readonly string _hubName;
         private readonly IServerNameProvider _nameProvider;
 
-        public ServiceHubDispatcher(IServiceProtocol serviceProtocol,
+        internal bool EnableGracefulShutdown

```

Comment:
why adding properties to the Dispatcher? Is it possible to fetch options from `_options` directly when needed?

Reply:
No, we can't, pls check how I use these properties in the related PR.